protos/ecs.proto

```diff
@@ -0,0 +1,37 @@
+//
+// Bentobox
+// Proto
+// ECS types
+//
+
+syntax = "proto3";
+package bento;
+
+import "types.proto";
+import "graph.proto";
+
+// EntityDef - registerable defintion for a entity in ECS
+message EntityDef {
+    // Unique id that identifies this entity
+    // This should be auto assigned by bentobox when first registered.
+    int64 id = 1;
```

Comment:
I think `uint64` would be even better since it is unlikely we will start entity ids from negative numbers

Reply:
How about uint32? I don't think  we can have more than 4 billion entities.